images/Dockerfile.kairos-debian

```diff
@@ -114,10 +113,17 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     zerofree \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
 
+###############################################################
```

Comment:
this change is not from this PR but from https://github.com/kairos-io/kairos/pull/3084/files

Reply:
rebase?